nurses_2/app.py

```diff
@@ -136,7 +138,11 @@ def run(self):
         except asyncio.CancelledError:
             pass
         finally:
-            print(defer_stderr.getvalue(), file=sys.stderr, end="")
+            if self.log_file:
+                with open(self.log_file, "w") as log:
+                    print(defer_stderr.getvalue(), file=log, end="")
```

Comment:
```py
if self.log_file:
    self.log_file.write_text(defer_stderr.getvalue())
```
Once a path, this method will be available.

Reply:
Ok, but this will force the user to cast to `Path` first